src/guide/debugging-models/logging.rst

```diff
@@ -0,0 +1,245 @@
+.. _DebuggingModelsLogging:
+
+Logging Timeseries Data
+=======================
+
+:ref:`Logging<Configuring Data to be Logged>` can be used to collect timeseries data, such as how the size of agent populations change throughout a model's execution. Furthermore, with :ref:`ensembles<ensembles>` this data can be collected across a batch of runs, especially useful when dealing with stochastic models.
+
+Use of logging in this manner is particularly useful when porting a pre-existing model to FLAME GPU 2, and is an approach we have used widely to validate our FLAME GPU 2 models behaviour matches.
+
+1. Decide a model configuration which can be executed in both versions of the model
+2. Execute both models ~100+ times, with a variety of different random seeds.
```

Comment:
Just state `many` rather than `~100+`? 

May also be worth stating that non-stochastic models don't need this (though they probably don't exist).